app/src/main/java/org/jellyfin/mobile/app/AppModule.kt

```diff
@@ -102,7 +106,20 @@ val applicationModule = module {
             }
         }
 
-        DefaultDataSource.Factory(context, baseDataSourceFactory)
+        val dataSourceFactory = DefaultDataSource.Factory(context, baseDataSourceFactory)
+
+        // Add authorization header. This is needed as we don't pass the
+        // access token in the url for Android Auto.
+        ResolvingDataSource.Factory(dataSourceFactory) { dataSpec: DataSpec ->
+            val authorizationHeaderString =
+                "MediaBrowser Token=\"${apiClient.accessToken}\", " +
+                    "Client=\"${apiClient.clientInfo.name}\", " +
+                    "Device=\"${apiClient.deviceInfo.name}\", " +
+                    "DeviceId=\"${apiClient.deviceInfo.id}\", " +
+                    "Version=\"${apiClient.clientInfo.version}\""
```

Comment:
The SDK has an AuthorizationHeaderBuilder utility for this that has proper escaping